modules/microshift-lvms-system-requirements.adoc

```diff
@@ -11,15 +11,18 @@ Using LVMS in {microshift-short} requires the following system specifications.
 [id="lvms-volume-group-name_{context}"]
 == Volume group name
 
-The default integration of LVMS selects the default volume group (VG) dynamically. If there are no volume groups on the {microshift-short} host, LVMS is disabled.
+If you did not configure LVMS in an `lvmd.yaml` YAML file placed in the `/etc/microshift/` directory, {microshift-short} attempts to assign a default volume group (VG) dynamically by running the `vgs` command. 
+* {microshift-short} assigns a default VG when only one VG is found. 
+* If more than one VG is present, the VG named `microshift` is assigned as the default. 
+* If a VG named `microshift` does not exist, LVMS is not deployed.
 
-If there is only one VG on the {microshift-short} host, that VG is used. If there are multiple volume groups, the group `microshift` is used. If the `microshift` group is not found, LVMS is disabled.
+If there are no volume groups on the {microshift-short} host, LVMS is disabled.
 
 If you want to use a specific VG, LVMS must be configured to select that VG. You can change the default name of the VG in the configuration file. For details, read the "Configuring the LVMS" section of this document.
 
 You can change the default name of the VG in the configuration file. For details, read the "Configuring the LVMS" section of this document.
 
-Prior to launching, the `lvmd.yaml` configuration file must specify an existing VG on the node with sufficient capacity for workload storage. If the VG does not exist, the node controller fails to start and enters a `CrashLoopBackoff` state.
+After the {microshift-short} starts, you can update the `lvmd.yaml` to include or remove VGs. To implement changes, you must restart {microshift-short}. If the `lvmd.yaml` is deleted, {microshift-short} attempts to find a default VG again.
```

Comment:
Tiny nit, otherwise this also LGTM
```suggestion
After {microshift-short} starts, you can update the `lvmd.yaml` to include or remove VGs. To implement changes, you must restart {microshift-short}. If the `lvmd.yaml` is deleted, {microshift-short} attempts to find a default VG again.
```